benchmarks/animation-bp/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.
+  module('animationBenchmark', ['ngAnimate'], config).
```

Comment:
<sub>Just in case it wasn't clear, I was just kidding (in an attempt to distract everyone from the fact that I was busted trying to sneak in those trailing dots 😛 </sub>